spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java

```diff
@@ -742,9 +742,9 @@ public void streamWithConfigServer() {
 				.put("app.*.management.endpoints.web.exposure.include", "*")
 				.put("app.*.spring.cloud.streamapp.security.enabled", "false");
 
-		// TODO this misterious 80 port set for K8s comes from the ATs and i'm not sure why. But without it partition tests fail.
 		if (this.runtimeApps.getPlatformType().equalsIgnoreCase(RuntimeApplicationHelper.KUBERNETES_PLATFORM_TYPE)) {
 			propertiesBuilder.put("app.*.server.port", "80");
```

Comment:
re: port 80 comment above, likely at some point somebody simply wanted to use port 80. the current AT's use 8080 which was only changed from 80 since there were some issues running in containers that had non-privileged access. at anyrate, no action needed as its simply a port to use. id be curious to know why partitions would fail without this, actually looks like in k8s deployer at least, if no `server.port` is defined, it defaults to 8080 anyway -- but feel free to ping on slack if wanting to discuss